services/web/src/components/CodeBlock.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
was this moved from docs? it only really applies to docs pages so kinda made sense that it was in there I think... alternately a `/docs/` folder inside components?

Reply:
I think it was in both places but regardless i am gonna use it outside docs ... was thinking it was better to only components in that case

Reply:
where else would we use it?

Reply:
for the applications / logs

Reply:
ok I see...maybe a `docs` subfolder in components makes sense then?

Reply:
what is the general plan with components i am not sure i know or follow the pattern for components

Reply:
Up to you but I see documentation as something that many users will likely rip out of a project as it's not needed so I think it makes sense to keep it in a separate folder at least. Arguably it should be a different app entirely but this is how we've set it up for now

Reply:
kk fair enough i might move it all back then

Reply:
yeah that's a good point 👍